KIPs/kip-fungible_token.md

```diff
@@ -0,0 +1,156 @@
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+The following standard allows for the implementation of a standard API for tokens within smart contracts.
+This standard provides basic functionality to transfer tokens.
+
+## Motivation
+<!--The motivation is critical for KIPs that want to change the Klaytn protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the KIP solves. KIP submissions without sufficient motivation may be rejected outright.-->
+A standard interface allows any tokens on Klaytn to be re-used by other applications: from wallets to decentralized exchanges.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Klaytn platforms (klaytn). -->
+This document derived heavily from Ethereum's [ERC-20 token standard](https://eips.ethereum.org/EIPS/eip-20) written by Fabian Vogelsteller and Vitalik Buterin.
+
+### Summary of Methods and Events
+The table below is a summary of methods.
+The prototype uses the syntax from Solidity `0.4.24` (or above).
+If the optional field is not marked, the function must be implemented.
+
+|Name|Optional|Prototype|
+|---|---|---|
+|[totalSupply](#totalsupply)| |function totalSupply() public view returns (uint256)|
+|[balanceOf](#balanceof)| |function balanceOf(address _owner) public view returns (uint256 balance)|
+|[transfer](#transfer)| |function transfer(address _to, uint256 _value) public returns (bool success)|
+|[name](#name)|O|function name() public view returns (string)|
+|[symbol](#symbol)|O|function symbol() public view returns (string)|
+|[decimals](#decimals)|O|function decimals() public view returns (uint8)|
+
+The table below is a summary of events.
+The prototype uses the syntax from Solidity `0.4.24` (or above).
+All the following events must be implemented.
+
+|Name|Prototype|
+|---|---|
+|[Transfer](#transfer-1)|event Transfer(address indexed _from, address indexed _to, uint256 _value)|
+
+### Methods
+
+#### totalSupply
+
+Returns the total token supply.
+
+```solidity
+function totalSupply() public view returns (uint256)
+```
+
+
+
+#### balanceOf
+
+Returns the balance of the account specified by the address `_owner`.
+
+```solidity
+function balanceOf(address _owner) public view returns (uint256 balance)
+```
+
+
+
+#### transfer
+
+Transfers `_value` amount of tokens to address `_to`, and MUST fire the [Transfer event](#transfer-1).
+The function SHOULD `throw` if the message caller's balance does not have enough tokens to spend.
+
+*Note* Transfers of 0 values MUST be treated as normal transfers and fire the [Transfer event](#transfer-1).
+
+```solidity
+function transfer(address _to, uint256 _value) public returns (bool success)
+```
+
+
+#### name
+
+Returns the name of the token - e.g. `"Klaytn"`.
```

Comment:
How about using other word like below. It may cause misunderstood with base token.
```suggestion
Returns the name of the token - e.g. `"MyToken"`.
```